.metadata

```diff
@@ -4,7 +4,27 @@
 # This file should be version controlled and should not be manually edited.
 
 version:
-  revision: cf4400006550b70f28e4b4af815151d1e74846c6
-  channel: stable
+  revision: "dec2ee5c1f98f8e84a7d5380c05eb8a3d0a81668"
```

Comment:
why these files are changed?

Reply:
Some changed maybe due my Ide compatibility.. Just as to run them